resources/guides/ta_guide.md

```diff
@@ -0,0 +1,99 @@
+- Each GA position is expected to be for a total of 140 hours. A limited number of GA positions may be available for fewer hours.
+- The total number of hours of a TA appointment will depend on the number of applications received and is anticipated to be 75-100 hours per position.  Note that Teaching Assistants can be employed for no more than 100 TA hours per semester total.
+
+
+Once confirming that you meet the basic requirements for eligibility:
+
+- Successful completion of the course (80% minimum average) or demonstrated experience/knowledge in subject matter (which will be determined by interview). 
+- Must have a minimum cumulative average of 75%. Must have a minimum major average of 80%. 
+- Must have excellent communication skills. Brightspace/Teams experience
+- Must be familiar with Brightspace and MS Teams or be able to acquire knowledge through training.  
+- Remote internet access requirements: Must have access to a computer with a reliable internet connection with video and audio capabilities. 
+    
+Or reviewing to see if you meet the preferred requirements:
+
+- Preferred qualifications will only be considered if there are 2 or more applicants who meet the required essential qualifications and who are relatively equal based on the criteria as per articles 12:03 or 12:04 of the CUPE 4580 collective agreement. 
+- Preference for TA positions will be given to applicants registered in a Computer Science program.  
+- Completion of CTL GA/TA Academy workshop (s) is an asset.
+- Centre for Teaching and Learning (CTL) Brightspace Training completed.
+
+You can begin the process by visiting the [School of Computer Science Online GA/TA Application website](https://forms.myweb.cs.uwindsor.ca/modules/form_builder/published/ta_ga_application.php)
+
+There are 5 pages to fill out, including your contact information, previous experiences, and top 10 courses ranked in order of your preferred placement.
+
+Once the form is submitted, if a successful candidate you will hear back from the University with an offer letter _typically_ 1-4 weeks before classes begin.
+
+One important thing to note is the University has the right to change your position if needed. This means if you are offered to TA for one course and sign the contract, they could simply send you an email saying you have been reassigned to another course instead.
+
+To accept a TA position there are a few steps involved
+
+1. Reply to the email offer with a message such as: "I accept this offer and confirm that I will be available to report for in-person duties in F/W/S term 20XX."
+2. Confirm you are enrolled in at least one course (This could even be a Coop course)
+3. [Digitally sign the contract](https://www.uwindsor.ca/graduate-studies/1236/adding-digital-signature-pdf-fillable-form)
+4. Complete the HR hiring package
+5. Ping a Mod in the CSS Discord to give you the TA Role! 
```

Comment:
I'm not sure if you're including this in 4. or not but if its your first time TAing you also need to get an employee number. I forget exactly what that looks like, I think you email HR or something.